assets/js/romo/selected_options_list.js

```diff
@@ -0,0 +1,185 @@
+var RomoSelectedOptionsList = function(focusElem) {
+  this.elem      = undefined;
+  this.focusElem = focusElem; // picker/select dropdown elem
+
+  this.items                  = [];
+  this.focusStyleClass        = this.elem.data('romo-option-list-focus-style-class');
+  this.defaultRmIconClass     = undefined;
+  this.defaultRmIconPaddingPx = 5;
+  this.defaultRmIconPosition  = 'right'
+  this.focusBlurTimeoutId     = undefined;
+
+  this.doInit();
+  this._bindElem();
+}
+
+RomoSelectedOptionsList.prototype.doInit = function() {
+  // override as needed
+}
+
+/*
+Options are specified as a list of items.  Each 'option' item object
+has display text and a value. They are compatible with the option list
+dropdown option items.
+
+Example:
+[ { 'type': 'option', 'value': 'a', 'displayText': 'A' },
+  { 'type': 'option', 'value': 'b', 'displayText': 'B' },
+  { 'type': 'option', 'value': 'c', 'displayText': 'C' }
+]
+*/
+
+RomoSelectedOptionsList.prototype.doSetItems = function(itemsList) {
+  this.items = itemsList;
+  this.doRefreshUI();
+}
+
+RomoSelectedOptionsList.prototype.doClearItems = function() {
+  this.items = [];
+  this.doRefreshUI();
+}
+
+RomoSelectedOptionsList.prototype.doAddItem = function(item) {
+  if (!this._getItemValues().includes(item.value)) {
+    this.items.push(item);
+    this.doRefreshUI();
+  }
+}
+
+RomoSelectedOptionsList.prototype.doRemoveItem = function(itemValue) {
+  var index = this._getItemValues().indexOf(itemValue);
+  if (index > -1) {
+    this.items.splice(index, 1);
+    this.doRefreshUI();
+  }
+}
+
+RomoSelectedOptionsList.prototype.doRefreshUI = function() {
+  var itemValues  = this._getItemValues();
+  var uiListElem  = this.elem.find('.romo-selected-option-list-items');
+  var uiItemElems = uiListElem.find('.romo-selected-option-list-item');
+  var uiValues    = uiItemElems.get().map(function(uiItemNode) {
+    return $(uiItemNode).data('romo-selected-option-list-value');
+  });
+  var rmNodes = uiItemElems.get().filter(function(uiItemNode) {
+     return itemValues.indexOf($(uiItemNode).data('romo-selected-option-list-value')) === -1;
+   });
+  var addItems = this.items.filter(function(item) {
+    return uiValues.indexOf(item.value) === -1;
+  });
+  rmNodes.each(function(rmNode) {
+    $(rmNode).remove();
+  });
+  addItems.each(function(addItem) {
+    uiListElem.append(this._buildItemElem(addItem));
+  });
+
+  var firstItemNode    = uiListElem.find('.romo-selected-option-list-item')[0];
+  var itemHeight       = parseInt(Romo.getComputedStyle(firstItemNode, "height"), 25);
```

Comment:
@kellyredding - I'm probably missing something but why are you using a radix of 25 with `parseInt`?